atc/worker/worker_helper.go

```diff
@@ -57,6 +59,13 @@ func (w workerHelper) createGardenContainer(
 			Env:        env,
 			Properties: gardenProperties,
 		})
+
+	metrics.
```

Comment:
This is web's perspective of how long a container creation is taking.

I wonder if this would be more valuable if we included the worker name in the label set, so that we could tell what's the particular worker that is facing some long time to have the container created 🤔  

Naturally, in the best scenario we'd have distributed tracing in a way that would allow us to have all the context and separate between time taken as seen from the `web` and time taken by just `garden`, but I think what we have for now (+ having worker name in the label set) would be good enough